minikube/deployment.yaml

```diff
@@ -0,0 +1,49 @@
+# deployment.yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: module-14
+spec:
+  selector:
+    matchLabels:
+      app: module-14
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: module-14
+    spec:
+      containers:
+        - name: todo-app
+          image: todo-app:prod
```

Comment:
I expected to also see `imagePullPolicy: Never` so it doesn't try to look for the image on Docker Hub.
This was suggested in the exercise instructions